scripts/build.py

```diff
@@ -1325,7 +1325,7 @@ def compile_typescript_files_continuously(project_dir):
     safe_delete_directory_tree(COMPILED_JS_DIR)
     python_utils.PRINT('Compiling ts files in watch mode...')
     cmd = [
-        './node_modules/typescript/bin/tsc', '--project', project_dir,
+        'node', './node_modules/typescript/bin/tsc', '--project', project_dir,
```

Comment:
This seems like it references a global command. Can we not do that and use our locally-installed version instead?

Reply:
This will use the local one, as commented above.

https://github.com/oppia/oppia/pull/8106#discussion_r352516294